CRM/Core/Form/RecurringEntity.php

```diff
@@ -409,7 +409,7 @@ public static function postProcess($params = [], $type, $linkedEntities = []) {
             }
           }
           //Ready to execute delete on entities if it has delete function set
-          if (CRM_Utils_Array::value('delete_func', CRM_Core_BAO_RecurringEntity::$_recurringEntityHelper[$params['entity_table']]) &&
+          if (!empty(CRM_Core_BAO_RecurringEntity::$_recurringEntityHelper[$params['entity_table']]['delete_func']) &&
             CRM_Utils_Array::value('helper_class', CRM_Core_BAO_RecurringEntity::$_recurringEntityHelper[$params['entity_table']])
```

Comment:
Ditto - there's another instance in the same `if` statement that could also be converted.

Reply:
Fixed.